### DISCUSSION TITLE
NER Use case - One token multiple entities

### DISCUSSION THREAD

**Discussion by Praful932** [explosion/spaCy#7009] [Help: Best practices]
I am working on a use case where I need to do NER like this

Lorem ipsum dolor sit amet, consectetur adipiscing elit
|---Entity 1--|
|-------Entity 2---- |

Is it possible to do NER in such a way or I am looking at it the wrong way? 
It would be great if anyone could point me in the right direction

Thanks

**Comment by ines** [comment]
A named entity recognition model typically predicts token-based tags, so by definition, you wouldn't be able to predict and represent overlapping or nested spans this way. So it probably makes more sense to think of what you're trying to do as a more general span prediction / recognition task.

We're currently working on a trainable `SpanRecognizer` component for this. See here for details: #6747 
Also see these related threads for more info and background: #4966, #5645, #6829

**Reply by Praful932** [reply]
Thanks, makes sense.
